### PR TITLE
[#12081] Fix rubric question overflow for instructors edit session page

### PR DESCRIPTION
**Pull request by jasonqiu212**
Part of #12081 
Sub-issue [Instructor edit sessions page: rubric question](https://github.com/TEAMMATES/teammates/projects/16#card-88348685)

**Outline of Solution**

- To add labels, I added `<thead>` and used `<th>` tags to specify the headers for a table, as per [WCAG guide](https://www.w3.org/WAI/tutorials/tables/two-headers/)
- To make table horizontal scrollable, I wrapped the table under a `table-responsive` tag
- To make left-most column maintain a readable width, I added a `min-width: 200px`
    - I could not use bootstrap elements for this, as bootstrap only supports `min-width` for `100vw`

### CODE REVIEW

File: src/web/app/components/question-types/question-edit-details-form/rubric-question-edit-details-form.component.html [hunk 1]
```diff
@@ -8,69 +8,73 @@
...
+              <button id="btn-delete-subquestion" type="button" class="btn btn-light" (click)="deleteSubQuestion(i)"
+                      (mouseover)="rowToHighlight = i" (mouseleave)="rowToHighlight = -1"
+                      [disabled]="model.rubricSubQuestions.length === 1 || !isEditable">
+                <i class="fas fa-times"></i>
+              </button>
+              <button type="button" class="btn btn-light" [disabled]="i === this.model.rubricSubQuestions.length - 1 || !isEditable"
+                      ngbTooltip="Move row down" (click)="moveRow(i, i + 1)">
+                <i class="fas fa-arrow-down"></i>
+              </button>
+            </div>
```
Comment (weiquu) [inline src/web/app/components/question-types/question-edit-details-form/rubric-question-edit-details-form.component.html:42]: Think can add `aria-label`s for these buttons? Else the screen reader just reads it as "button". If I'm not wrong, shouldn't be too complicated, just use the same text as in the `ngbTooltip` / "Delete row".

File: src/web/app/components/question-types/question-edit-details-form/rubric-question-edit-details-form.component.html [hunk 2]
```diff
@@ -8,69 +8,73 @@
...
               <i class="fas fa-times"></i>
             </button>
-            <button type="button" class="btn btn-light" [disabled]="i === this.model.rubricSubQuestions.length - 1 || !isEditable"
-                    ngbTooltip="Move row down" (click)="moveRow(i, i + 1)">
-              <i class="fas fa-arrow-down"></i>
+            <button type="button" class="btn btn-light" [disabled]="j === this.model.rubricChoices.length - 1 || !isEditable"
+                    ngbTooltip="Move column right" (click)="moveChoice(j, j + 1)">
+              <i class="fas fa-arrow-right"></i>
             </button>
           </div>
```
Comment (weiquu) [inline src/web/app/components/question-types/question-edit-details-form/rubric-question-edit-details-form.component.html:72]: Same comment as the previous buttons!

File: src/web/app/components/question-types/question-edit-details-form/rubric-question-edit-details-form.component.html [hunk 3]
```diff
@@ -8,69 +8,73 @@
...
+            <textarea class="form-control min-vw-50" [ngModel]="model.rubricSubQuestions[i]"
+                      (ngModelChange)="triggerRubricSubQuestionChange($event, i)" rows="3" [disabled]="!isEditable"></textarea>
+          </div>
+        </th>
+        <td *ngFor="let rubricDescriptionCell of rubricDescriptionRow; let j = index; trackBy: trackByIndex" [ngClass]="{'highlight-deletion': j === columnToHighlight}">
+          <textarea class="form-control" [ngModel]="rubricDescriptionCell"
+                    (ngModelChange)="triggerRubricDescriptionChange($event, i, j)" rows="3" [disabled]="!isEditable"></textarea>
+          <input *ngIf="model.hasAssignedWeights" type="number" class="form-control margin-top-10px" step="0.01"
+                 [disabled]="!isEditable"
+                 [ngModel]="model.rubricWeightsForEachCell[i][j]" (ngModelChange)="triggerRubricWeightChange($event, i, j)">
```
Comment (weiquu) [inline src/web/app/components/question-types/question-edit-details-form/rubric-question-edit-details-form.component.html:52]: Can add a simple `aria-label` for the input here, e.g. "Choice Weight Input"

**Review by weiquu — CHANGES_REQUESTED** [review]
Looks good (: If not too complicated, would be worth adding the `aria-labels` mentioned for better screen reader experience

**Review by weiquu — APPROVED** [review]
LGTM!

**Review by zhaojj2209 — APPROVED** [review]

**Pull request merged**